sklearn/preprocessing/tests/test_label.py

```diff
@@ -513,3 +513,17 @@ def test_inverse_binarize_multiclass():
                                                    [0, 0, 0]]),
                                        np.arange(3))
     assert_array_equal(got, np.array([1, 1, 0]))
+
+
+def test_empty_labels_transform():
+    le = LabelEncoder()
+    le.fit(np.array([1, 2, 1, 2, 2], dtype=np.float64))
```

Comment:
Please use str in the array so that the tests fail in master and success in current PR.

Reply:
Got it